### PR TITLE
Release 2021 05 14

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

File: speech-to-text/v1.ts [hunk 1]
```diff
@@ -279,6 +279,7 @@ namespace SpeechToTextV1 {
     splitTranscriptAtPhraseEnd?: boolean;
     speechDetectorSensitivity?: number;
     backgroundAudioSuppression?: number;
+    lowLatency?: boolean;
```
Comment (dpopp07) [inline speech-to-text/v1.ts:282]: This needs to be accompanied by changes in `lib/recognize-stream.ts` in order for the new parameter to actually be supported

File: assistant/v1.ts [hunk 2]
```diff
@@ -5198,9 +5336,11 @@ namespace AssistantV1 {
     agent_unavailable?: AgentAvailabilityMessage;
     /** Routing or other contextual information to be used by target service desk systems. */
     transfer_info?: DialogNodeOutputConnectToAgentTransferInfo;
+    /** An array of objects specifying channels for which the response is intended. */
+    channels?: ResponseGenericChannel[];
   }
 
-  /** An object that describes a response with response type `image`. */
+  /** DialogNodeOutputGenericDialogNodeOutputResponseTypeImage. */
```
Comment (dpopp07) [inline assistant/v1.ts:5343]: It's interesting that we're seeing these changes from an actual description to what looks like an ID. It doesn't seem very desirable to me. This is coming from the generator, right? I wonder if the descriptions were removed from the API definition.

It only seems to be happening in Assistant (v1 and v2)

File: test/integration/assistant.v1.test.js [hunk 3]
```diff
@@ -1066,4 +1066,20 @@ describe('assistant v1 integration', () => {
...
+        input: {
+          text: 'Turn on the lights',
+        },
+        workspaceId: options.workspaceId,
+      };
+
+      const res = await assistant.bulkClassify(params);
+      const { result } = res || {};
+      expect(result).toBeDefined();
+      console.log(result);
```
Comment (dpopp07) [inline test/integration/assistant.v1.test.js:1082]: Is this console log intentional? Or leftover from testing?

**Review by dpopp07 — CHANGES_REQUESTED** [review]
Looks good for the most part. The hand-written websocket code needs to be updated and and I left a couple other comments about things that looked unusual

File: speech-to-text/v1.ts (hunk 1 above)
Comment (apaparazzi0329) [reply speech-to-text/v1.ts:282]: Thank you, I did not realize this

File: test/integration/assistant.v1.test.js (hunk 3 above)
Comment (apaparazzi0329) [reply test/integration/assistant.v1.test.js:1082]: Good catch that will be removed

File: assistant/v1.ts (hunk 2 above)
Comment (apaparazzi0329) [reply assistant/v1.ts:5343]: Yes looking at the previous version of the api refs it looks like the description for these discriminator models was removed. As to why, I would not know but it had to be done on purpose

Comment (dpopp07) [reply assistant/v1.ts:5343]: Yeah that's weird. Nothing that should hold up the release though

**Pull request merged**